<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CALIDAD-08" ignored="0" name="mavenProject" tests="3" failures="1" timestamp="16 ago 2018 15:55:30 GMT" time="83.902" errors="0">
  <testcase name="LoginCorrecto" time="7.523" classname="mavenProject.testng"/>
  <testcase name="LoginSinContrasena" time="4.55" classname="mavenProject.testng"/>
  <testcase name="LoginSinUsuario" time="36.695" classname="mavenProject.testng">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at mavenProject.testng.LoginSinUsuario(testng.java:74)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- LoginSinUsuario -->
</testsuite> <!-- mavenProject -->
